Mathlib/Algebra/Hom/Centroid.lean

```diff
@@ -542,7 +542,8 @@ def star_ring (mul_comm : ∀ f g : CentroidHom α, f * g = g * f) : StarRing (C
   star_mul := fun f g => by
     rw [mul_comm]
     ext
-    rw [star_apply, mul_apply, mul_apply, star_apply, star_apply, star_star]
+    rw [star_apply, mul_apply, mul_apply, star_apply]
+    simp
```

Comment:
In the lean3 version of this PR https://github.com/leanprover-community/mathlib/pull/18096/files#diff-b9e451cbb3a5b6efa050e64d3f4896b5d3ed3c51e9f034259709f8372aaaa525R342 I could put
```
rw [star_apply, mul_apply, mul_apply, star_apply, star_apply, star_star]
```
But doing this in Mathlib4 results in:
```
 Build: Mathlib/Algebra/Hom/Centroid.lean#L538
(deterministic) timeout at 'whnf', maximum number of heartbeats (200000) has been reached (use 'set_option maxHeartbeats <num>' to set the limit)

```

Reply:
Looks like this is resolved by now.

Reply:
The PR has changed a lot in the last 18 months(!) but I think the equivalent line is: https://github.com/leanprover-community/mathlib4/pull/6595/files#diff-4c5d1ef43cba4fe158b5e0ad5afbba8f622bd9479cbb954544f3526f15ae999fR121